neurolib/utils/devutils.py

```diff
@@ -1,40 +1,391 @@
 import matplotlib.pyplot as plt
 import numpy as np
+import scipy
 
 import neurolib.utils.functions as func
 
 
-def plot_outputs(model, bold_transient=10000):
-    _, axs = plt.subplots(2, 3, figsize=(12, 6))
+def plot_outputs(
```

Comment:
I use this file only for my personal plotting purposes etc....

Reply:
wunderbar! I won't review it then :)